metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/BaseUserMetadataService.java

```diff
@@ -59,53 +57,68 @@ public void saveMetadata(final String userId, final HasMetadata holder, final bo
                 saveDataMetadata(dataDto.getDataUri(), userId, Optional.of(newMetadata), merge);
             }
         }
+
+        if (holder instanceof BusinessMetadata) {
```

Comment:
If business metadata is stored as JSON, then why dont we store it in definition metadata and just apply rules?